### ISSUE TITLE
Just a Few Fish Mod not working as should

### ISSUE THREAD

**Issue by XcaliFlame** [8bit-Build/Peace-of-Mind#9]
I am trying to complete the quest Gotta Catch 'Em All in The Basics quest tab but when I fish up a salmon or cod the fish does not go inside of the bucket and goes straight into my inventory

**Comment by chunkedelement** [comment]
Could be a bug issue or conflict with just a few fish mod will review  thanks

**Comment by chunkedelement** [comment]
Hey are you using the just a few fish: wood, Iron, gold, or Diamond fishing rods? The vanilla rod will not work. Just tested and they are all working. Can you verify? Thanks
Pack version: 2.1.0
JAFF Mod version: 1.7 for 1.12.2

**Comment by XcaliFlame** [comment]
I've been using the diamond rod and the fish go into my inventory instead
of the water bucket

**Comment by chunkedelement** [comment]
Does the iron one work? All of the fish or just some? I tested with the iron and gold and it worked, is it biome dependent? The mod dev doesn’t have a github that I can find to post questions, are you using optifine at all?

**Comment by XcaliFlame** [comment]
Nope, I found another way to get the fish though, I just went around and
found out I can right click the fish swimming to put them in the bucket, I
don't know why the normal way doesn't work

**Comment by chunkedelement** [comment]
Thanks for testing, I tried with cod and the others and they all went in the bucket using iron and wood rods, bucket was in main displayed inventory slot.

**Comment by chunkedelement** [comment]
Ok cool, will reach out to others to see if this a known issue, not sure if the mod is being maintained still by dev, a peaceful recipe may be in the works if can’t figure out a root cause

**Comment by chunkedelement** [comment]
Tried on client and server can’t reproduce this issue so far will continue to check.

**Comment by chunkedelement** [comment]
When you tried fishing did the bucket have water in it first? I did baiting the rod with seeds as well and the fish will go in a water bucket (all 4 types) with water filled first, in main inventory slots. Not sure if this is a isolated issue as asked other players and the fishing into a bucket from JAFF mod is working for them as well.

**Comment by XcaliFlame** [comment]
Yeah there was water in the bucket

**Comment by chunkedelement** [comment]
ok thanks figured but wanted to be sure

**Comment by chunkedelement** [comment]
Unmodified version of the 2.1.0 pack without any mod additions correct? optifine or other mods.

**Comment by XcaliFlame** [comment]
I wasn't using any additional mods other than the ones in the base pack

**Comment by chunkedelement** [comment]
Thanks just wanted to verify as sometimes optifine can randomly cause visual bugs. Seems sporadic this issue some players it works and a few have this issue, will see if dev has input.

**Comment by chunkedelement** [comment]
Mod dev twitch tracker: https://www.curseforge.com/minecraft/mc-mods/just-a-few-fish/issues/27

**Comment by chunkedelement** [comment]
Not sure if this is fixable currently as it’s hard to pinpoint the specific root cause or possible Mod conflict without data or logs of some kind.

**Comment by chunkedelement** [comment]
Last update Apr 18, 2020, mod likely EOL due to 1.12.2.
